examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -411,10 +417,37 @@ def act(self, obs, env):
         human_pos = env._sim.agents_mgr[
             1 - self._agent_idx
         ].articulated_agent.base_transformation.translation
+        human_rot = env._sim.agents_mgr[
+            1 - self._agent_idx
+        ].articulated_agent.base_rot
         robot_pos = env._sim.agents_mgr[
             self._agent_idx
         ].articulated_agent.base_transformation.translation
 
+        # Check if the human walks or rotates
+        is_human_walk = (
+            np.linalg.norm(
+                np.array((self._prev_human_pos - human_pos))[[0, 2]]
+            )
+            > 0.01
+            if self._prev_human_pos is not None
+            else False
+        )
+        is_human_rotate = (
```

Comment:
Do you think it's necessary to resume social nav if the human is rotating without changing translation? E.g. this would be relevant if our social-nav policy tries to face the front of the human (or if it might do this in the future).

Reply:
For my observation based on few cases, it seems that it does not matter. However, it is safer to let robots move so that it can adjust its orientation. And by the way, good point! Happy to discuss more :)